test/fluree/db/query/fql_test.clj

```diff
@@ -109,35 +109,53 @@
                               [?s :ex/favNums ?favNum]
                               [?s :schema/name ?name]
                               [?s :schema/age ?age]]
-                    :values [[?email ?favNum] [["alice@example.org" 42]
-                                               ["cam@example.org" 37]]]}]
+                    :values  [[?email ?favNum] [["alice@example.org" 42]
+                                                ["cam@example.org" 37]]]}]
             (is (= [["Alice" 50]]
                    @(fluree/query db q))
                 "returns only the results related to the existing bound values")))))))
 
+(deftest ^:integration bind-query-test
+  (let [conn   (test-utils/create-conn)
+        people (test-utils/load-people conn)
+        db     (fluree/db people)]
+    (testing "bind query w/ fn value"
+      (let [q '{:context {:ex "http://example.org/ns/"}
+                :select  [?firstLetterOfName ?name ?decadesOld]
+                :where   [[?s :schema/age ?age]
+                          {:bind {?decadesOld (quot ?age 10)}}
+                          [?s :schema/name ?name]
+                          {:bind {?firstLetterOfName (subStr ?name 0 1)}}]
```

Comment:
I think it might be useful to have a test that has a bind map with multiple entries. Maybe even combine the two here (after the pattern involving `?name`).

Reply:
![giphy](https://user-images.githubusercontent.com/59748/220681547-a0480ee3-e2b0-4497-9477-639b9815c253.gif)